src/lib/KeyboardTypes.js

```diff
@@ -0,0 +1,110 @@
+const KEYBOARD_TYPES = {
+  CURSIVE: 'cursive',
+  MINISCULE: 'miniscule',
+  SQUARE: 'square'
+};
+
+const KeyboardOptions = [
+  {
+    name: 'Ashkenazi Cursive',
+    type: KEYBOARD_TYPES.CURSIVE,
+    img: '../images/keyboards/ashkenazi-cursive.jpg'
```

Comment:
Did I mention how I really, really like how you're organising the script characters into a consolidated image file instead of having one .jpg file _per character?_ will++, becky++